### PR TITLE
change test runner to work on windows

### PR DESCRIPTION
**Pull request by dirk-thomas**
Depends on ament/ament_cmake#7.

Connects to ament/ament_cmake#7

### CODE REVIEW

**Comment by dirk-thomas** [comment]
@esteve @tfoote @wjwwood Please review.

**Comment by wjwwood** [comment]
Lgtm, but why was so much code removed in `ament_copyright`? Was that left over from the separating of them into separate Python and CMake packages?

**Comment by dirk-thomas** [comment]
Ups, the copyright stuff was not intended to be in that commit. I updated the PR to exclude the copyright stuff.

**Comment by wjwwood** [comment]
+1

File: ament_cmake_copyright/cmake/ament_copyright.cmake [hunk 1]
```diff
@@ -19,15 +19,14 @@ function(ament_copyright)
     message(FATAL_ERROR "ament_copyright() could not find program 'ament_copyright'")
   endif()
 
-  set(subcmd "${ament_copyright_BIN} --xunit-file \"${AMENT_TEST_RESULTS_DIR}/${PROJECT_NAME}/${ARG_TESTNAME}.xml\"")
-  foreach(arg ${ARG_UNPARSED_ARGUMENTS})
-    set(subcmd "${subcmd} \"${arg}\"")
-  endforeach()
+  set(cmd "${ament_copyright_BIN}" "--xunit-file" "${AMENT_TEST_RESULTS_DIR}/${PROJECT_NAME}/${ARG_TESTNAME}.xml")
```
Comment (tfoote) [inline ament_cmake_copyright/cmake/ament_copyright.cmake]: No PYTHON_EXECUTABLE on this line? All the others have it.

Comment (dirk-thomas) [inline ament_cmake_copyright/cmake/ament_copyright.cmake]: No, it is an extension point which will generate an executable on Windows.

**Comment by esteve** [comment]
+1

**Comment by tfoote** [comment]
+1

**Pull request merged**